src/app/clients/edit/[id]/autofill.ts

```diff
@@ -38,7 +38,10 @@ const autofill = (
         numberOfChildren: children.length,
         children: children,
         listType: clientData.default_list,
-        dietaryRequirements: arrayToBooleanGroup(clientData.dietary_requirements ?? []),
+        dietaryRequirements:
+            clientData.dietary_requirements !== null
+                ? arrayToBooleanGroup(clientData.dietary_requirements)
+                : null,
         feminineProducts: arrayToBooleanGroup(clientData.feminine_products ?? []),
```

Comment:
It would be better for all these fields that are nullable in the DB to be handled properly here - feminine, pet, other

Reply:
Making that change would require more logic in presenting and handling the null / Don't Know case in the UI for each thing, so have decided to leave as-is